underlay/src/main/resources/config/datamapping/sd/entity/weight/all.sql

```diff
@@ -9,7 +10,6 @@ SELECT
     WHEN xvw.x_invalid = 'N' THEN 'Clean'
     ELSE 'Raw'
   END AS is_clean_name,
-  CASE WHEN xvw.x_invalid = 'N' THEN TRUE ELSE FALSE END x_invalid_bool,
```

Comment:
Did you mean to remove this?

Reply:
It possibly was removed from previous sd directory. I checked the ENT_weight in both indexed_sd20230831_beta and indexed_sd20230831 (the previous indexed dataset) and the coldefs match as well as row counts.  And the counts form the UI in my local matches that seen in sd-discover. I updated the excel doc with my findings